test/functional/p2p_addrv2_relay.py

```diff
@@ -74,6 +74,16 @@ def run_test(self):
 
         assert addr_receiver.addrv2_received_and_checked
 
+        self.log.info("Check that the node sends a SENDADDRV2 message to outbound full relay peers")
+        conn = self.nodes[0].add_outbound_p2p_connection(P2PInterface(), p2p_idx=0)
+        conn.sync_with_ping()
```

Comment:
```suggestion
        conn.sync_send_with_ping()
```
to prevent intermittent test failures due to `SendMessages` not being called in time (same below).